src/components/form.jsx

```diff
@@ -0,0 +1,41 @@
+import { useDispatch } from 'react-redux';
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/book/book';
+
+const Form = () => {
+  const [title, setTitle] = useState('');
+  const [author, setAuthor] = useState('');
+  const dispatch = useDispatch();
+  const addNewBook = (e) => {
+    e.preventDefault();
+    const newBook = {
+      item_id: uuidv4(),
+      title,
+      author,
+      category: 'Action',
+    };
+    dispatch(addBook(newBook));
+    setAuthor('');
+    setTitle('');
+  };
+
+  const titleInput = (e) => {
+    setTitle(e.target.value);
+  };
+
+  console.log(title);
```

Comment:
- [ ] Good job so far, but I think it would be better if you delete this line. `console.log` kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website 🙃 🙃